Python Nightly build failure

It looks like the `skipif` version is not defined correctly, causing this test to fail instead of being skipped on certain older Python versions.

```
2023-04-16T04:14:23.9339581Z =================================== FAILURES ===================================
2023-04-16T04:14:23.9339907Z _______________ TestType.test_implicit_convert_from_builtin_type _______________
2023-04-16T04:14:23.9340452Z 
2023-04-16T04:14:23.9340590Z self = <test_type.TestType object at 0xeb8a6d78>
2023-04-16T04:14:23.9340764Z 
2023-04-16T04:14:23.9340962Z     @pytest.mark.skipif(sys.version_info < (3,8), reason="requires >=python3.8")
2023-04-16T04:14:23.9341329Z     def test_implicit_convert_from_builtin_type(self):
2023-04-16T04:14:23.9341629Z >       type = duckdb.list_type(list[str])
2023-04-16T04:14:23.9342156Z E       TypeError: 'type' object is not subscriptable
2023-04-16T04:14:23.9342343Z 
2023-04-16T04:14:23.9342483Z /project/tests/fast/test_type.py:80: TypeError
2023-04-16T04:14:23.9342788Z =========================== short test summary info ============================
2023-04-16T04:14:23.9343182Z FAILED ../../../project/tests/fast/test_type.py::TestType::test_implicit_convert_from_builtin_type
2023-04-16T04:14:23.9343555Z ================== 1 failed, 667 passed, 18 skipped in 23.93s ==================
2023-04-16T04:14:24.6958621Z ##[endgroup]

```

--- a/tools/pythonpkg/tests/fast/test_type.py
+++ b/tools/pythonpkg/tests/fast/test_type.py
@@ -75,7 +75,7 @@ class TestType(object):
         assert str(type) == 'UNION(a BIGINT, b VARCHAR, c TINYINT)'
 
     import sys
-    @pytest.mark.skipif(sys.version_info < (3,8), reason="requires >=python3.8")
+    @pytest.mark.skipif(sys.version_info < (3,9), reason="requires >= python3.9")
     def test_implicit_convert_from_builtin_type(self):
         type = duckdb.list_type(list[str])
         assert str(type.child) == "VARCHAR[]"

